holidata/holidays/hr-HR.py

```diff
@@ -7,18 +7,19 @@
 class hr_HR(Locale):
     """
     01-01: [NF] Nova Godina
-    01-06: [NRF] Bogojavljanje / Sveta tri kralja
+    01-06: [NRF] Bogojavljanje ili Sveta tri kralja
     05-01: [NF] Praznik rada
     06-22: [NF] Dan antifašističke borbe
     06-25: [NF] Dan državnosti
-    08-05: [NF] Dan domovinske zahvalnosti
+    08-05: [NF] Dan pobjede i domovinske zahvalnosti i Dan hrvatskih branitelja
     08-15: [NRF] Velika Gospa
     10-08: [NF] Dan neovisnosti
-    11-01: [NRF] Dan svih svetih
+    11-01: [NRF] Svi sveti
+    11-18: [NRF] Dan sjećanja na žrtve Domovinskog rata i Dan sjećanja na žrtvu Vukovara i Škabrnje
```

Comment:
If this is a new holiday, which is valid from 2020 on, we have to add it as a function instead of putting it into the docopt:
```
def holiday_dan_sjecanja_na_zrtve_Domovinskog_rata_i_Dan_sjecanja_na_zrtvu_Vukovara_i_Skabrnje(self):
    if self.year >= 2020:
        return [Holiday(
            self.locale,
            "",
            SmartDayArrow(self.year, 11, 18),
            "Dan sjećanja na žrtve Domovinskog rata i Dan sjećanja na žrtvu Vukovara i Škabrnje",
            "NF"
        )]
    else:
        return []

```

Reply:
> If this is a new holiday, which is valid from 2020 on, we have to add it as a function instead of putting it into the docopt:
> 
> ```
> def holiday_dan_sjecanja_na_zrtve_Domovinskog_rata_i_Dan_sjecanja_na_zrtvu_Vukovara_i_Skabrnje(self):
>     if self.year >= 2020:
>         return [Holiday(
>             self.locale,
>             "",
>             SmartDayArrow(self.year, 11, 18),
>             "Dan sjećanja na žrtve Domovinskog rata i Dan sjećanja na žrtvu Vukovara i Škabrnje",
>             "NF"
>         )]
>     else:
>         return []
> ```

Correct, this is a new one, valid from 2020.

Reply:
I will add the necessary function after merging your PR